client/src/components/base/ProjectForm.tsx

```diff
@@ -53,7 +59,8 @@ function ProjectForm({ currentGrantId }: { currentGrantId?: string }) {
     }
 
     await dispatch(saveFileToIPFS(formInputs, FileTypes.PROJECT));
-    dispatch(publishGrant(currentGrantId));
+    await dispatch(publishGrant(currentGrantId));
+    setTimeout(() => navigate(slugs.grants, { replace: true }), 1500);
```

Comment:
Added a timeout so the project created notification shows for 1.5 seconds before they are redirected

Reply:
@schultztimothy  cool! do we need the `replace` there?

Reply:
Good call! I removed both instances of replace